use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,238 @@
+# Using Intel GPU device with Kata Containers
+
+Support for using Intel GPU devices with Kata Containers has been added in  Kata Containers version 1.3.0.
+
+An Intel Graphics device can be passed to a Kata Containers container using GPU passthrough (Intel GVT-d)
+as well as using hardware base GPU sharing (Intel GVT-g)
+
+Intel GVT-d (one VM to one physical GPU) also named as Intel-Graphics-Device
+pass-through feature is one flavor of graphics virtualization approach.<br/>
+This flavor allows direct assignment of an entire GPU’s prowess to a single user,
+passing the native driver capabilities through the hypervisor without any limitations.
+
+ Intel GVT-g (multiple VMs to one physical GPU) is a full GPU virtualization solution
+ with mediated pass-through.<br/>
+ A virtual GPU instance is maintained for each VM, with part of performance critical
+ resources, directly assigned.The capability of running native graphics driver inside a VM, without hypervisor intervention in performance critical paths, achieves a good balance among performance, feature, and sharing capability.
```

Comment:
Missing space ` ` after full stop `assigned.The`.

Reply:
done.